src/integrationTest/java/uk/gov/hmcts/reform/judicialapi/elinks/BaseLocationsIntegrationTest.java

```diff
@@ -62,12 +62,7 @@ void test_elinksService_load_base_location_return_status_200() {
 
         List<BaseLocation> baseLocationList = baseLocationRepository.findAll();
 
-        assertEquals(1, baseLocationList.size());
-        assertEquals("0", baseLocationList.get(0).getBaseLocationId());
-        assertEquals("default", baseLocationList.get(0).getCourtName());
-        assertEquals("default", baseLocationList.get(0).getCourtType());
-        assertEquals("default", baseLocationList.get(0).getCircuit());
-        assertEquals("default", baseLocationList.get(0).getAreaOfExpertise());
+        assertEquals(0, baseLocationList.size());
```

Comment:
Since removed the base_location_id from base_location_type table so the size will 0 hence removed above assertion from the test case same followed for th ebelow on.